client/src/components/Category/index.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import "./style.css";
+import axios from "axios";
+class Category extends React.Component {
+  state = {
+    result: []
+  };
+  componentDidMount() {
+    axios
+      .get("/category")
+      .then(data => {
+        return data;
+      })
+      .then(data => {
+        const resultArray = data.data.data.map(e => e);
+        this.setState({ result: resultArray });
+      });
+  }
+
+  render() {
+    return (
+      <div className="grid-container">
+        {this.state.result.map(e => (
```

Comment:
It would more of user friendly if you add a condition to render a progress bar or "Loading..." while the waiting for the response form the server side... something like 
```
this.state.result?(this.state.map(e=>...):<h2>Loading</h2>
```

Reply:
i add loading in page